service/resource/configmap/configmap.go

```diff
@@ -0,0 +1,63 @@
+package configmap
```

Comment:
Stubbing here as the operatorkit framework shits the bed if not given any resources. Next PR is filling this out